backend/src/routers/gameRouter.ts

```diff
@@ -53,6 +53,43 @@ const addGameSchema = z.object({
 	playtime: z.number().int().min(1)
 });
 
+/**
+ * @api {get} /api/v1/games/searcg/:term Request Games
+ * @apiName SearchGames
+ * @apiGroup Games
+ * @apiDescription Get all public games that includes the search term
+ *
+ * @apiSuccess {Object[]} games List of games
+ *
+ * @apiSuccessExample Success-Response:
+ *  HTTP/1.1 200 OK
+ *  [
+ *   {
+ *    "id": "clgkri8kk0000przwvkvbyj95",
+ *    "name": "Game 1",
+ *    "description": "Game 1 description",
+ * 	  "platformName": "Steam",
+ *	  "releaseDate": "2023-04-13",
+ *	  "playtime": "60"
+ *   }
+ * ]
+ */
+ gameRouter.get('/search', async (req, res) => {
```

Comment:
Shouldn't this be a post request?

Reply:
I don't think so since we only query data, we don't do any processing or modification on the server